### DISCUSSION TITLE
关于 分享 说明

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#115] [Announcement]
SAI 设计初衷是清单下载器，因为由于Steam启用了清单请求码，导致 清单 在无相关账号的情况下变得很难获取。  
后期加了很多方便的功能，但是功能加的越多越便捷，就越容易被坏心思的人利用和滥用，违背开心游戏的原则。  

所以 为了让自己少一些糟心，SAI 默认功能回归本质

##  软件功能说明
  + 清单下载
  + 根据Steam已有的密钥匹配清单，对游戏和DLC写入解锁配置

## 如何分享
在 [密钥清单分享](https://github.com/pjy612/SteamManifestCache/discussions/categories/Share) 讨论板块，分享 密钥 或 该仓库未收录的 密钥+清单 即可。  
分享时 烦请参考 顶置的 分享模板帖子

**Comment by pcz88188** [comment]
无法获取最新版本的sai

**Reply by pjy612** [reply]
项目已关闭

**Comment by Luochengfeihua** [comment]
请问获取内测资格还需要什么条件呀，我分享了似乎没有收到过邮件，是还需要什么资格吗

**Reply by pjy612** [reply]
分享是为了内测？没有内测，请不要分享，谢谢
